### DISCUSSION TITLE
Numjuck & include subdirectories (for e.g. {% include 'home/header" %})?

### DISCUSSION THREAD

**Discussion by denik1981** [11ty/eleventy#1949] [Q&A] [answered]
Is it possible to reference subidirectories in the include statement? I'm not being able to reference my partial by using this form but it picks it up when "header" is in the root f the "_includes" folder.

This is the basic file structure I have to test this.   
I'm prototyping the problem so to extract just the part I'm interesting in.  

Tree
---
```
_includes
---- home
------ header.njk

_layouts
---- home.njk
```
.eleventy.js
---
```
dir.layourts: '_layouts'
templateFormats: ['html', 'njk', 'md'],
```
__inlcude_ is taking the default value here.

home.njk
---
```
{% include 'home/header' %} 
```

Console
---
```
TemplateContentRenderError` was thrown
Error: template not found: header.njk
```

I have also tried with 
- "home/header" (double quotes)
- home/header.njk (unquotes)
- Using relative paths ('./home/header') & replacing the "home" subfolder inside "layouts".

Is this possible to do? What am I doing wrong?

**Comment by pdehaan** [comment, marked as answer]
Does it work if you use `{% include "home/header.njk" %}`?

https://github.com/pdehaan/11ty-1949

**Reply by denik1981** [reply]
Yes, it does. 
I guess I was confussed by a liquid example (not njk) where the extension was not needed.  
Thanks for the fast answer!
